src/dotnetTestExplorer.ts

```diff
@@ -111,21 +111,33 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
         }
     }
 
+    private enableBuild(): string {
+        const dotnetBuildOption = Utility.getConfiguration().get<boolean>("build");
+        return dotnetBuildOption ? "--no-build" : "";

```

Comment:
A little confusing: When `dotnet-test-explorer.build` is true, adding `--no-build`?